components/places/src/db/schema.rs

```diff
@@ -64,6 +64,13 @@ const CREATE_TABLE_HISTORYVISITS_SQL: &str =
         FOREIGN KEY(from_visit) REFERENCES moz_historyvisits(id)
     )";
 
+const CREATE_TABLE_HISTORYVISIT_TOMBSTONES_SQL: &str = "CREATE TABLE moz_historyvisit_tombstones (
+        place_id INTEGER NOT NULL,
+        visit_date INTEGER NOT NULL,
+        FOREIGN KEY(place_id) REFERENCES moz_places(id) ON DELETE CASCADE,
+        UNIQUE(place_id, visit_date)
```

Comment:
Why not a `PRIMARY KEY`?

Reply:
One's created automatically and we don't need it. Initially it was WITHOUT ROWID but that complained. I'll add it to avoid a case where we regret it not being there.

Reply:
@thomcc Oh, sorry, I meant, why not make this `PRIMARY KEY(place_id, visit_date)` instead? It seems like you always look up by those, and then you wouldn't need the indices?

Reply:
Ah, yep, this lets us ditch the other indices too, thanks!